File: chatwoot/python/src/ContactCreateExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
    # api_key={"agentBotApiKey": "AGENT_BOT_API_KEY"},
    # api_key={"platformAppApiKey": "PLATFORM_APP_API_KEY"},
)

with ApiClient(configuration) as api_client:
    contact_create = models.ContactCreate(
        inbox_id=None,
        name=None,
        email=None,
        phone_number=None,
        avatar_url=None,
        identifier=None,
        avatar=None,
    )

    try:
        response = api.ContactsApi(api_client).contact_create(
            account_id=None,
            contact_create=contact_create,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling ContactsApi#contact_create: %s\n" % e)

File: chatwoot/ruby/src/ContactCreateExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
    # config.api_key["api_access_token"] = "AGENT_BOT_API_KEY"
    # config.api_key["api_access_token"] = "PLATFORM_APP_API_KEY"
end

contact_create = ChatwootClient::ContactCreate.new
contact_create.inbox_id = (0).to_f

begin
    response = ChatwootClient::ContactsApi.new.contact_create(
        0, # account_id
        contact_create, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling ContactsApi#contact_create: #{e}"
end
